simulations/vip-254/simulations.ts

```diff
@@ -0,0 +1,86 @@
+import { expect } from "chai";
+import { BigNumber } from "ethers";
+import { formatUnits, parseEther, parseUnits } from "ethers/lib/utils";
+import { ethers } from "hardhat";
+
+import { initMainnetUser, setMaxStaleCoreAssets } from "../../src/utils";
+import { NORMAL_TIMELOCK, forking, testVip } from "../../src/vip-framework";
+import vip254 from "../../vips/vip-254/bscmainnet";
+import COMPTROLLER_ABI from "./abi/Comptroller.json";
+import IERC20_ABI from "./abi/IERC20Abi.json";
+import VTOKEN_ABI from "./abi/VBep20DelegateAbi.json";
+
+const USDC = "0x8AC76a51cc950d9822D68b83fE1Ad97B32Cd580d";
+const VUSDC = "0xecA88125a5ADbe82614ffC12D0DB554E2e2867C8";
+const CHAINLINK_ORACLE = "0x1B2103441A0A108daD8848D8F5d790e4D402921F";
+
+const TREASURY = "0xF322942f644A996A617BD29c16bd7d231d9F35E9";
+const LIQUIDATE_AND_REDEEM_HELPER = "0xA08301b7C5f4BccD654De95E8C9BD4388CC54Ec1";
+const VBNB = "0xA07c5b74C9B40447a954e1466938b865b6BBea36";
+const LIQUIDATOR = "0x0870793286aaDA55D39CE7f82fb2766e8004cF43";
+const COMPTROLLER = "0xfD36E2c2a6789Db23113685031d7F16329158384";
+
+const EXPLOITER_WALLET = "0x489A8756C18C0b8B24EC2a2b9FF3D4d447F79BEc";
+const BINANCE_WALLET = "0x6657911F7411765979Da0794840D671Be55bA273";
+const REPAY_AMOUNT = parseUnits("39137.18", 18);
+const EXPECTED_BNB_AMOUNT = parseUnits("131.130431007685944719", 18);
```

Comment:
Constants can be reused by importing from the VIP implementation.

Reply:
Idk, I still prefer to not import implementation to tests, it kinda allows me to notice the exact tests that break when I change the impl